test/user.test.js

```diff
@@ -0,0 +1,245 @@
+const request = require('supertest');
+const expect = require('chai').expect;
+
+const app = require('../index');
+const fixture = require('../fixtures/user');
```

Comment:
You will change this code, but this is a function and its name should tell what it does, which doesn't happen in this case.